dist/yaml/ovnkube.yaml

```diff
@@ -59,8 +60,8 @@ spec:
           readOnly: true
         # Mount the entire run directory for socket access for Docker or CRI-o
         # TODO: remove
-        - mountPath: /var/run
-          name: host-var-run
+        #- mountPath: /var/run
```

Comment:
I think we can just remove the mapping of /var/run.  ovnkube doesn't use docker or CRIO directly like openshift-sdn did.

Reply:
I agree, no need to mount the parent.